region/maridia/inner-yellow.json

```diff
@@ -829,51 +829,70 @@
                   "notable": false,
                   "requires": [
                     "Morph",
-                    "h_canNavigateUnderwater",
                     {"or":[
                       "h_canBombThings",
                       "h_canUseSpringBall"
+                    ]},
+                    {"or":[
+                      "HiJump",
+                      "h_canFly",
+                      "canWalljump",
+                      "canSpringBallJumpMidAir",
+                      "canTrickyDashJump",
+                      "canTrickyUseFrozenEnemies"
                     ]}
                   ]
                 },
                 {
-                  "name": "Maridia Bug Room Frozen Menu Bridge",
-                  "notable": true,
+                  "name": "Maridia Bug Room MidAir Morph with Gravity",
+                  "notable": false,
                   "requires": [
                     "Morph",
-                    "h_canNavigateUnderwater",
-                    "canTrickyUseFrozenEnemies"
+                    "Gravity",
+                    "canTrickyJump",
+                    {"or":[
+                      "HiJump",
+                      "h_canFly",
+                      "canWalljump",
+                      "canSpringBallJumpMidAir",
+                      "canTrickyDashJump",
+                      "canTrickyUseFrozenEnemies"
+                    ]}
                   ],
                   "note": [
-                    "Freeze one of the bugs (called Menus) in a position where you can use it to get into the morph passage.",
-                    "That allows passage without having to gain height while being morphed."
+                    "With Gravity, it's possible to move forward and mid-air morph into the morph passage.",
+                    "The momentum can push Samus into the tunnel, no need to press forward while in mid-air."
                   ]
                 },
                 {
                   "name": "Maridia Bug Room Suitless MidAir Morph",
                   "notable": false,
                   "requires": [
                     "h_canNavigateUnderwater",
```

Comment:
canNavigateUnderwater could be removed here too.  You're waiting for the water to go away, not using it.